litho-widget/src/main/java/com/facebook/litho/widget/RenderInfoViewCreatorController.java

```diff
@@ -25,31 +25,55 @@
   @VisibleForTesting
   final SimpleArrayMap<ViewCreator, Integer> mViewCreatorToViewType = new SimpleArrayMap<>();
 
+  public static final int DEFAULT_COMPONENT_VIEW_TYPE = 0;
+  private final boolean mCustomViewTypes;
+  private final int mComponentViewType;
+  private int mViewTypeCounter;
 
-  public static final int COMPONENT_VIEW_TYPE = 0;
-  private int mViewTypeCounter = COMPONENT_VIEW_TYPE + 1;
+  public RenderInfoViewCreatorController(boolean hasCustomViewType, int componentViewType) {
+    mCustomViewTypes = hasCustomViewType;
```

Comment:
Same comment for these names as the earlier one, can we rename these to `mCustomViewTypeEnabled`/`customViewTypeEnabled`?